packages/foam-core/src/core.ts

```diff
@@ -0,0 +1,89 @@
+import { Graph, Edge } from 'graphlib'
```

Comment:
Maybe call this file (core.ts) as notegraph.ts?

Reply:
👍 -- I'd say `note-graph.ts` since capital G makes it a compound word, and therefore should be separated.